spec/index.html

```diff
@@ -1074,14 +1071,14 @@ <h3>Algorithm</h3>
                       <a>identifier issuer</a> in result.</li>
                     <li id="hndq-5-4-5-5">If <var>chosen path</var> is not empty and the length
                       of <var>path</var> is greater than or equal to the length
-                      of <var>chosen path</var> and <var>path</var> is <a>lexicographically</a>
-                      greater than <var>chosen path</var>, then skip to the next
-                      <var>permutation</var>.</li>
+                      of <var>chosen path</var> and <var>path</var> is
+                      greater than <var>chosen path</var> when consider <a>code point order</a>,
```

Comment:
```suggestion
                      greater than <var>chosen path</var> when considering <a>code point order</a>,
```

Alternatively, we could ```<var>path</path> is <a>code point ordered</a> greater than``` ... right?